src/features/ModelPlan/ModelToOperations/_components/Table/index.tsx

```diff
@@ -492,6 +493,7 @@ const MTOTable = () => {
 
   return (
     <>
+      <MTOTableActions />
```

Comment:
I hesitate to but the component within `<MTOTable />`.  It can be moved as a sibling inside of the home component.  This allows some flexibility and decoupling of the components if they want to be used independently elsewhere

```
{currentView === 'milestones' && <><MTOTableActions /><MTOTable />< />}
```